examples/ads.amp.html

```diff
@@ -265,6 +266,12 @@
       data-host="talos.adman.gr">
   </amp-ad>
 
+  <h2>AdmanMedia</h2>
+  <amp-ad width="300" height="250"
```

Comment:
When testing locally, as the mona.admanmedia.com does not set specially CORS headers for localhost, I am launching Chrome like this: ` ./chrome --disable-web-security --user-data-dir `
Prior to launching it to production we will setup CORS headers on our server and this error will not be launched anymore